bingus-bot/src/contexts/reply.ts

```diff
@@ -40,10 +54,37 @@ export const replyContext: ContextMenu = {
             .setColor("#65459A")
             .setFooter({ text: `${data[0].relevance.toFixed()}% relevant` })
             .data,
+
+        ],
+        components: [showB],
+      });
+
+      const collector = message.createMessageComponentCollector();
+
+      collector.on('collect', async i => {
+        interaction.deleteReply();
+        interaction.targetMessage.reply({
+        embeds: [
+          new EmbedBuilder()
+            .setAuthor({
+              name: `Triggered by ${interaction.user.displayName}`,
+              iconURL: interaction.user.avatarURL() ?? undefined,
+            })
+            .setTitle(data[0].title)
+            .setDescription(data[0].text)
+            .setColor("#65459A")
+            .setFooter({text: `${data[0].relevance.toFixed()}% relevant` })
+
+            
+            .data,
+
         ],
       });
+      });
```

Comment:
just use await instead of doing a component collector: `await message.awaitMessageComponent()`